tests/test_usage.py

```diff
@@ -382,6 +382,11 @@ def test_except1(self):
         exc = en.except1([88, 77, 21, 66, 79, 77])
         assert exc.to_list() == [4, -5, 25, 12, 0]
 
+    def test_except1_unhashable(self):
+        lst = [[1, 2, 3], [4, 5], [6, 7], [6, 7]]
+        spec_lst = [[4,], [1, 2]]
+        assert Enumerable(lst).select(lambda lst: lst[:-1]).except1(spec_lst).to_list() == [[6]]
```

Comment:
this is except test do prefer not calling select...

stick to the format of existing code